### DISCUSSION TITLE
Audio/Microphone access  in web app

### DISCUSSION THREAD

**Discussion by metador** [flet-dev/flet#893] [Ideas]
Hi ,

Wanted to check if there is a way to get to audio devices or audio input so an app can get user voice commands.
I did see a side thread for this but not sure if how to use it.
https://github.com/flet-dev/flet/issues/419

Sincerely,

**Comment by ndonkoHenri** [comment]
Subscribe to that issue/thread.
It is not yet implemented/available, but when it will be, you will get notified from there.
